src/infoview.ts

```diff
@@ -69,8 +69,8 @@ export class InfoProvider implements Disposable {
                 this.rerender();
             }),
             commands.registerCommand('_lean.revealPosition', this.revealEditorPosition),
-            commands.registerCommand('_lean.hoverPosition', (u, l, c) => { this.hoverEditorPosition(u, l, c); }),
-            commands.registerCommand('_lean.stopHover', () => { this.stopHover(); }),
+            // commands.registerCommand('_lean.hoverPosition', (u, l, c) => { this.hoverEditorPosition(u, l, c); }),
```

Comment:
If you remove the `_lean.hoverPosition` command, then please delete it completely.  Do not just comment it out.